packages/snaps-controllers/src/test-utils/multichain.ts

```diff
@@ -0,0 +1,111 @@
+import type { PermissionConstraint } from '@metamask/permission-controller';
+import { SnapEndowments } from '@metamask/snaps-rpc-methods';
+import { SnapCaveatType } from '@metamask/snaps-utils';
+import { MOCK_SNAP_ID } from '@metamask/snaps-utils/test-utils';
+import type { CaipAccountId, CaipChainId, Json } from '@metamask/utils';
+
+export const BTC_CAIP2 =
+  'bip122:000000000019d6689c085ae165831e93' as CaipChainId;
```

Comment:
Nit: We also have some defined scopes in the `keyring-api` now (in case you'd like to use them):
- https://github.com/MetaMask/accounts/blob/main/packages/keyring-api/src/btc/constants.ts#L6-L12

NOTE: Since CAIP is becoming quite present everywhere now, I wonder if there would be a better place to put those kind of constant, the keyring API does feel always appropriate... and I don't think `@metamask/utils` is a good fit either. Anyway, topic for later 🙃

Reply:
IMO not that important for testing purposes, but I agree they could be moved to utils and used easier here. I don't think bringing in the `keyring-api` here is appropriate.

Reply:
Yup, I do agree with the `keyring-api`, having them somewhere else would be better. Thanks!